packages/@modelfusion-llamacpp-bindings/update_submodules.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I tend to put helper scripts into the `bin` subfolder, e.g. `packages/@modelfusion-llamacpp-bindings/bin/update_submodules.sh` would follow that convention here.